client/src/components/OwnerProfileComponents/OwnerDetails.js

```diff
@@ -6,19 +6,111 @@ import { useStoreContext } from "../../utils/GlobalState";
 import { UPDATE_CURRENT_USER } from "../../utils/actions";
 import { cities, neighbourhoods } from '../../utils/helpers';
 
-function OwnerDetails(props) {
-    const {
-        results,
-    } = props
+function OwnerDetails({ user }) {
 
-    function handleFormSubmit() {
-        console.log('do nothing')
-    }
+    // const {
+    //     user
+    // } = props
+
+    const [updateOwnerProfile, { error }] = useMutation(UPDATE_OWNER_PROFILE);
+    const [state, dispatch] = useStoreContext();
+
+    const [formData, setFormData] = useState({ 
+        first_name: '', 
+        last_name: '', 
+        email: '',
+        address_city: ''
+    });
+    
+    useEffect(() => {
+        if (user) {
+            const { first_name, last_name, email, address, address_city, status } = user;
+            if(status === "PENDING_INFORMATION") {
```

Comment:
What is the purpose of this if statement?
Both true and false conditions render the same action?